### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by trimethoxy**
Add signature

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -162,8 +162,6 @@ Signed,
 
 [Daniel](https://github.com/Zuzuk-null)
 
-[Stanislav Shmarov](https://github.com/Snarpix)
-
```
Comment (nekonee) [inline index.md:166]: Please don't delete other contributors.

File: index.md [hunk 2]
```diff
@@ -202,4 +200,4 @@ Signed,
 
 [karmich](https://github.com/karmich)
 
-[Jonny Lindqvist](https://github.com/ojnny)
```
Comment (nekonee) [inline index.md:205]: Don't delete other contributors.

**Review by nekonee — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (trimethoxy) [reply index.md:166]: Sorry, seems like github bug

**Comment by podvornyakva** [comment]
Please. add a file instead of edit index.md

Follow README.md instructions.

Thank you.

**Pull request closed without merging**